app/controllers/api/v1/accounts/credentials_controller.rb

```diff
@@ -10,8 +10,9 @@ def show
   end
 
   def update
-    current_account.update!(account_params)
     @account = current_account
+    @account.update!(account_params)
+    ActivityPub::UpdateDistributionWorker.perform_async(@account.id)
```

Comment:
- Can't be in a callback, callbacks execute in-transaction (queued job might get old data)
- Can't be a service because while this controller uses `update!`, the preferences controller uses `if update` (different usage)

With only two usages, it's easier to just have this one extra line in both places.